### PR TITLE
[Delivers 90103790]: Adds title and description field to ncr work orders

### PR DESCRIPTION
**Pull request by cmc333333**
* Adds the mentioned fields
* Saves/edits the *work order*'s name field, rather than the cart's

These changes were originally made when I thought NCR's field display would need to be different than other clients. The mechanics are in this PR, but turned out to not be required.
* Moves `client_partial` into its own helper
* Modified client partial to look for a `default` client
* Moves proposal_properties to this default client

Looks like:
screen shot 2015-03-27 at 2 48 57 pm
screen shot 2015-03-27 at 12 14 41 pm

### CODE REVIEW

File: app/controllers/ncr/work_orders_controller.rb [hunk 1]
```diff
@@ -77,16 +73,12 @@ def redirect_if_cart_cant_be_edited
     def permitted_params
       fields = Ncr::WorkOrder.relevant_fields(
         params[:ncr_work_order][:expense_type])
-      params.require(:ncr_work_order).permit(*fields)
+      params.require(:ncr_work_order).permit(:name, *fields)
```
Comment (afeld) [inline app/controllers/ncr/work_orders_controller.rb]: We don't want the `:name` in the `relevant_fields`?

File: app/helpers/client_helper.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Contains functions which make client selection/branching easier
```
Comment (afeld) [inline app/helpers/client_helper.rb]: :+1:

**Comment by afeld** [comment]
Hooray! I tweaked some Cart-related stuff in the WorkOrder model (and merged myself before yours – sorry!)...mind updating there, and also making the `description` available in the API?

> Saves/edits the work order's name field, rather than the cart's

Ooh, good step in moving towards removing the dependence on Carts!

File: app/controllers/ncr/work_orders_controller.rb (hunk 1 above)
Comment (cmc333333) [inline app/controllers/ncr/work_orders_controller.rb]: Name is special; it's never displayed with the other fields.

**Comment by cmc333333** [comment]
Updated, thanks

**Pull request merged**